### PR TITLE
SL-3774 Add post content objects

### PR DESCRIPTION
**Pull request by sergioebx**
### Description of Changes
- Add content-related objects for LinkedIn `Post` schema

### Documentation

[Post schema](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/posts-api#post-schema)

## Final Checklist

Please tick once completed.

- [x] Build passes.
- [ ] Versioning considered (the version number in this PR is inline with semantic 
versioning requirements).
- [ ] Change log has been updated.

### CODE REVIEW

File: src/main/java/com/echobox/api/linkedin/types/posts/MultiImageContentMedia.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import com.echobox.api.linkedin.types.urn.URN;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * MultiImageContentMedia object
+ *
+ * @author Sergio Abplanalp
+ */
+public class MultiImageContentMedia {
```
Comment (djjashton) [inline src/main/java/com/echobox/api/linkedin/types/posts/MultiImageContentMedia.java:30]: Do we need a MultiImageContentMedia and MediaContent?

File: src/main/java/com/echobox/api/linkedin/types/posts/TargetEntity.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+import java.util.List;
+
+/**
+ * Target Entity
+ * @see <a href="https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/posts-api#targetentities">Target Entities</a>
+ *
+ * @author Sergio Abplanalp
+ */
+public class TargetEntity {
+  // CPD-OFF
```
Comment (djjashton) [inline src/main/java/com/echobox/api/linkedin/types/posts/TargetEntity.java:36]: Would it be better to set the CPD-OFF in the older ones so it gets automatically removed when the old API is removed?

**Review by djjashton — APPROVED** [review]
Looks good. Just some minor comments that do not stop merge.

File: src/main/java/com/echobox/api/linkedin/types/posts/MultiImageContentMedia.java (hunk 1 above)
Comment (sergioebx) [reply src/main/java/com/echobox/api/linkedin/types/posts/MultiImageContentMedia.java:30]: Good catch, will rather use `MediaContent` in the multiple places

File: src/main/java/com/echobox/api/linkedin/types/posts/TargetEntity.java (hunk 2 above)
Comment (sergioebx) [reply src/main/java/com/echobox/api/linkedin/types/posts/TargetEntity.java:36]: Good suggestion, will make the change

**Pull request merged**